### PR TITLE
Added file on take()

### LINKED ISSUE #5598: [Term Entry] PyTorch Tensor Operations .take()
### Reason/inspiration (optional)

We would like a new entry on the `.take()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/take/take.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Neema-Joju**
### Description

Added a new entry file on .take(). Improving documentation and providing more context for the users to understand.

### Issue Solved

Closes #5598 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/take/take.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+Title: '.take()'
+Description: 'Returns a new tensor from a given tensor.'
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/take/take.md:3]:
```suggestion
Description: 'Returns a 1D tensor containing elements from input at the specified indices.'
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/take/take.md:3]: This can be elaborated

File: content/pytorch/concepts/tensor-operations/terms/take/take.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - 'AI'
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+---
+
+The **.take()** function returns a tensor from the given tensor. The resulting tensor will always be a 1-D tensor irrespective of the size of the given tensor.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/take/take.md:17]:
```suggestion
The **`.take()`** function returns a 1D tensor containing elements from the given tensor at the specified indices. The resulting tensor will always be a 1D tensor irrespective of the size of the given tensor. 
```

File: content/pytorch/concepts/tensor-operations/terms/take/take.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+The **.take()** function returns a tensor from the given tensor. The resulting tensor will always be a 1-D tensor irrespective of the size of the given tensor. 
+
+## Syntax
+
+```psuedo
+torch.take(input,index)
+```
+
+-`input`: the input tensor from which the elements will be taken.
+-`index`: the position of the elements you want to extract.
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/take/take.md:26]:
```suggestion
-`input`: The input tensor from which the elements will be selected.
-`index`: A 1D tensor containing the indices of the elements to extract from `input`.
```

File: content/pytorch/concepts/tensor-operations/terms/take/take.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Example
+
+The following example demonstrates the usuage of `.take()` method:
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/take/take.md:30]:
```suggestion
The following example demonstrates the usage of `.take()` function:
```

File: content/pytorch/concepts/tensor-operations/terms/take/take.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+```py
+import torch
+
+#Define a tensor
+data = torch.tensor([[4, 2, -1], [7, 8, 0]])
+
+#Define a new tensor
+torch.take(data, torch.take([0,2,5]))
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/take/take.md:39]:
```suggestion
import torch

# Define a tensor
data = torch.tensor([[4, 2, -1], [7, 8, 0]])

# Define indices as a tensor
indices = torch.tensor([0, 2, 5])

# Use torch.take with data and indices
result = torch.take(data, indices)

print(result)
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/take/take.md:39]: There is an error in the code due to a misuse of `torch.take()` in the `indices` argument. The correct code should use a tensor for the `indices` argument in `torch.take`, not calling `torch.take` again inside it.

File: content/pytorch/concepts/tensor-operations/terms/take/take.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+# Define a tensor
+data = torch.tensor([[6, 8, 1],
+                    [5, 2, 4]])
+
+# Calculate the transpose of the tensor
+out = torch.take(ten)
+
+print(out)
+```
+
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/take/take.md:62]:
```suggestion
```

Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/take/take.md:62]: This can be removed, we only need ## Example block for this entry

File: content/pytorch/concepts/tensor-operations/terms/take/take.md (hunk 3 above)
Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/take/take.md:26]: We avoid using pronouns like "you" and "we"

File: content/pytorch/concepts/tensor-operations/terms/take/take.md (hunk 4 above)
Comment (mamtawardhani) [reply content/pytorch/concepts/tensor-operations/terms/take/take.md:30]: Since we referred to as a 'function' earlier, let us stick to that

File: content/pytorch/concepts/tensor-operations/terms/take/take.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+---
+
+The **.take()** function returns a tensor from the given tensor. The resulting tensor will always be a 1-D tensor irrespective of the size of the given tensor. 
+
+## Syntax
+
+```psuedo
```
Comment (mamtawardhani) [inline content/pytorch/concepts/tensor-operations/terms/take/take.md:21]:
```suggestion
```pseudo
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Neema-Joju thank you for contributing to Codecademy Docs! 😄 

I've suggested some modifications, could you please review and make those changes at your earliest convenience? Thank you! 😃

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to Codecademy Docs @Neema-Joju 😄 

The entry looks good for a second round of review! 🚀

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @Neema-Joju, The entry is nicely written, it LGTM to get it merged. Thank you for contributing to Codecademy Docs :)

Thanks,
Savi

**Comment by Neema-Joju** [comment]
@SaviDahegaonkar 
Is there any way to be listed as a contributor instead of anonymous? I have updated my profile with my github details. It didn't seem to work.
